java/ql/test/library-tests/frameworks/JaxWs/JaxRsFlow.ql

```diff
@@ -12,6 +12,8 @@ class TaintFlowConf extends TaintTracking::Configuration {
   override predicate isSink(DataFlow::Node n) {
     exists(MethodAccess ma | ma.getMethod().hasName("sink") | n.asExpr() = ma.getAnArgument())
   }
+
+  override int fieldFlowBranchLimit() { result = 3 }
```

Comment:
Might as well set it to something large for this test given that we've found it to be a factor.
```suggestion
  override int fieldFlowBranchLimit() { result = 1000 }
```